static/test.html

```diff
@@ -2,17 +2,6 @@
 <head>
   <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta http-equiv="Content-Security-Policy"
-    content="default-src 'unsafe-inline';
-             style-src 'self' 'unsafe-inline';
-             font-src 'unsafe-inline';
-             script-src 'self' 'unsafe-inline';
-             worker-src blob: ;
-             child-src blob: ;
-             img-src 'unsafe-inline' data: blob: ;
-             media-src 'unsafe-inline';
-             connect-src https://*.tiles.mapbox.com https://api.mapbox.com https://events.mapbox.com 'unsafe-inline'"
-    >
```

Comment:
Is that security meta tag really unnecessary?

Reply:
No, it's not, let's not loose that. I tried using an iframe to integrate the FAQ at first and this tag blocked me from doing it. At first I was annoyed, but then I agreed.

Reply:
That page is only used for integration tests. Testcafe injects Javascripts which couldn't be loaded in first tries because of Content Security Policy. So I tried to set all values to "unsaf-inline" but realized later that it's sufficient to set the Content Security Policy in the headers: https://github.com/deltachat/deltachat-desktop/pull/1363/files#diff-7b1ed8e005428717d211bea7f7e5c1feR166
The content security for usage in production mode is not affected at all...

Reply:
Sorry, I didn't look at the filename properly... You're right.